src/common_subset.rs

```diff
@@ -12,7 +12,7 @@ use agreement::{Agreement, AgreementMessage};
 use broadcast;
 use broadcast::{Broadcast, BroadcastMessage};
 use fmt::HexBytes;
-use messaging::{DistAlgorithm, Target, TargetedMessage};
+use messaging::{DistAlgorithm, MessageIter, OutputIter, Target, TargetedMessage};
```

Comment:
It shouldn't be necessary to import the iterator types, since `DistAlgorithm` has convenience methods for them.